test/verify/machineslib.py

```diff
@@ -1445,6 +1461,12 @@ def fill(self):
                 b.click("#start-vm") # TODO: fix this, do not assume initial state of the checkbox
             # b.set_checked("#start-vm", self.start_vm)
 
+            if self.storage_pool != 'Automatic':
```

Comment:
Can you add a line to check that if Automatic is set for pool the volume entry is absent?